sql/moz-fx-data-shared-prod/snowflake_migration/dataset_metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Snowflake Migration
+description: |-
+  Queries for modeling raw snowplow data about prospects
+  (candidate links to be curated by ML models and the editorial team for New Tab).
+
+user_facing: false
```

Comment:
This failed to deploy because the dataset doesn't match the expectations (if it's non-user-facing, it must be suffixed with `_derived` or `_external`). We should probably catch this in CI.